Fixing filter by issue (#3415)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

  Delete and Discard is available on the Business plan.
</Alert>

> When you are unable to take immediate action on an issue, but it continues to occur, Sentry supports deleting and discarding that issue from the issue details page. Navigate to the issue you would like to filter, then click on the drop-down menu next to the bin icon, and select the “delete and discard future events” option. This setting deletes most data associated with the issue and filters out matching events before they count against your quota.

Reviewer: Original phrasing "...deleting from the issue details page" could be be understood to mean the effect is only taking place in the UI and not in the system generally. Reworded slightly.

```suggestion
When you are unable to take immediate action on an issue, but it continues to occur, Sentry supports deleting and discarding that issue, which you can do from the **Issue Details** page. Navigate to the issue you would like to filter, click on the drop-down menu next to the bin icon, and select “Delete and discard future events”. This setting deletes most data associated with the issue and filters out new matching events before they count against your quota.
```
